schedule/yast/autoyast_bcache.yaml

```diff
@@ -0,0 +1,81 @@
+---
+name: autoyast_bcache
+description: |
+  Test installation with AY profile for bcache with encrypted lvm.	
+  One contains bios boot partition, /boot and not formatted native linux partition.
+  Bcache uses not formatted native linux partition as backing device and second disk as a whole as caching device.
+vars:
+  NUMDISKS: 2
+  AUTOYAST_PREPARE_PROFILE: 1
+  AUTOYAST: autoyast_sle15/autoyast_bcache.xml
+schedule:
+  - autoyast/prepare_profile
+  - installation/isosize
+  - installation/bootloader_start
+  - autoyast/installation
+  - autoyast/console
+  - autoyast/login
+  - autoyast/clone
+  - autoyast/repos
+  - autoyast/logs
+  - autoyast/autoyast_reboot
+  - installation/grub_test
+  - installation/first_boot
+  - console/verify_bcache_partitioning
+  - console/validate_file_system
```

Comment:
we could schedule `validate_file_system` before `verify_bcache_partitioning` as it is more generic, so if the file system is ok we go on.

Reply:
thats correct but for technical reasons i put that order. If I change validate_file_system to work here i have to VR the other jobs that they use it

Reply:
I'm not following you here, if you just switch the order or those two you would not affect any other test suite and as far as you have some root console you would not need to run the VR because would probably work. Could you clarify?

Reply:
Maybe this will be more helpful http://aquarius.suse.cz/tests/1916#

Reply:
To achieve that order, you need `select_console 'root-console'`, just rebase your code, I already added it in another PR.

Reply:
order is resolved